pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/PostStageTest.java

```diff
@@ -111,6 +113,7 @@ public void withAllLocalUnsuccessfulWithNotBuilt() throws Exception {
         env(s).put("MAKE_RESULT", Result.NOT_BUILT.toString()).set();
         expect(Result.NOT_BUILT, "unsuccessful")
                 .logContains("I LOVE YOU VIRGINIA")
+                .logContains("I FAILED YOU, SORRY")
```

Comment:
I added this to make it clear what the actual behavior is, but do we want the unsuccessful condition to fire if the result is `NOT_BUILT`?